modules/nwtc-library/CMakeLists.txt

```diff
@@ -79,8 +79,8 @@ if(BUILD_OPENFAST_SIMULINK_API)
 
   if (APPLE OR UNIX)
     if (${CMAKE_Fortran_COMPILER_ID} STREQUAL "GNU")
-      # set(NWTC_SYS_FILE src/SysMatlabLinuxGnu.f90)
-      message(FATAL_ERROR "SIMULINK with GNU is not yet supported.")
+      set(NWTC_SYS_FILE src/SysMatlabLinuxGnu.f90)
+      # message(FATAL_ERROR "SIMULINK with GNU is not yet supported.")
```

Comment:
Since we are supporting this combination, can we remove this error message?

Reply:
Makes sense to me. No need to leave it as a comment.

Reply:
Thank you @bjonkman :)

Reply:
done